config/routes.rb

```diff
@@ -19,8 +19,10 @@
     resources :users, only: :destroy
     post "logout" => "user_sessions#destroy", :as => :logout
 
+    get '/flickr_search/search_flickr', to: 'flickr_search#search_flickr', as: :search_flickr
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.